HPS/Heartland/etc/frontend/routes.xml

```diff
@@ -20,5 +20,14 @@
         <route id="hpspaypal" frontName="hpspaypal">
             <module name="HPS_Heartland" />
         </route>
+        <route id="orderreview" frontName="hpsorder">
+            <module name="HPS_Heartland" />
+        </route>
+        <route id="cancelorder" frontName="hpsordercancel">
+            <module name="HPS_Heartland" />
+        </route>
+        <route id="completeorder" frontName="hpsordercomplete">
+            <module name="HPS_Heartland" />
+        </route>
```

Comment:
These shouldn't be their own frontnames. They should fall under the `hpspaypal` frontname.

To be honest, we should only have one frontname, `heartland`, but that can be an improvement outside of this PR.